### PR TITLE
Explore landing bug

### PR DESCRIPTION
**Pull request by mmeigs**
### Stories/Links:

DOP-4030

https://docs.google.com/spreadsheets/d/1XCmWfCCegeHUcAv6UFCquUM4sHxEhxjxZ5wPZ3NyCuo/edit#gid=0
Issue on Line 18

### Current Behavior:

https://mongodbcom-cdn.website.staging.corp.mongodb.com/docs-qa/

### Staging Links:

https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/newhomepagewhodis/landing/matt.meigs/explore-landing-bug/index.html

### Notes:

Easiest way to see the bug is to enbiggen your screen or zoom out. All other content holds to the a certain max width determined by the grid. It happens to be 1440px for the content. The Explore the Document Model content was not respecting this sort of max-width.

### CODE REVIEW

**Review by seungpark — COMMENTED** [review]
this LGTM. good catch on using container for full width background vs container for content! think we can still use flexbox here vs grid, but purely preference.

left some minor cleanups for hardcoded heights/widths!

File: src/components/Landing/Explore/index.js [hunk 1]
```diff
@@ -6,39 +6,50 @@ import { palette } from '@leafygreen-ui/palette';
...
+  column-gap: 63px;
+  min-height: 337px;
+
+  @media ${theme.screenSize.upToLarge} {
+    flex-direction: row;
   }
 `;
 
+const HeaderStyle = styled.div`
+  width: 329px;
```
[the remark this answers is not in the document]
Comment (mmeigs) [reply src/components/Landing/Explore/index.js:48]: I think Anabella was trying to ensure it didn't wrap. Oh, `text-wrap` would be cleaner, yeah

[the remark this answers is not in the document]
Comment (mmeigs) [reply src/components/Landing/Explore/index.js:48]: ignore above. it's better without.

**Comment by mmeigs** [comment]
@seungpark I merged and cleaned up more styles, encompassing your feedback and also padding more faithful to the figma. As for flex vs grid, I mainly wanted to keep it consistent with the other content on the page. But yes, I love flexbox so much more :) 
New staging link: https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/newhomepagewhodis/landing/matt.meigs/explore-landing-bug/index.html

File: src/components/Landing/Explore/index.js [hunk 2]
```diff
@@ -6,36 +6,40 @@ import { palette } from '@leafygreen-ui/palette';
...
+const ExploreItem = styled.div`
   @media ${theme.screenSize.xLargeAndUp} {
     background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="308" height="337" viewBox="0 0 308 337" fill="none"><path d="M183.5 11.2773L183.5 11.2907C183.875 39.1797 204.911 61.9519 230.994 61.9519H322.506C347.71 61.9519 368 84.0794 368 111.024V260.048C368 314.013 327.418 358 277.119 358C226.813 358 185.869 314.402 185.869 260.048V209.976C185.869 182.064 164.447 159.301 138.375 159.301H46.494C21.6488 159.301 1 137.561 1 110.229V11.6747C1 -15.2539 21.6438 -37 46.494 -37H138.006C162.866 -37 183.5 -15.6405 183.5 11.2773Z" stroke="%2300ED64" margin-left="865px" stoke-width="2"/></svg>');
     background-position: right;
     background-repeat: no-repeat;
   }
   background-color: ${palette.black};
+  grid-column-start: 1;
+  grid-column-end: 15;
+  padding: 56px 0;
```
Comment (seungpark) [inline src/components/Landing/Explore/index.js:18]: can we reduce on mobile?

seems like its being doubled down

image

Comment (mmeigs) [reply src/components/Landing/Explore/index.js:18]: Hmm, but that button gross wrapping is bad.

Comment (mmeigs) [reply src/components/Landing/Explore/index.js:18]: Oh, need to change the grid on mobile.

**Comment by mmeigs** [comment]
@seungpark new staging link lol 
https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/newhomepagewhodis/landing/matt.meigs/explore-landing-bug/index.html

File: src/components/Landing/Explore/index.js (hunk 2 above)
Comment (mmeigs) [reply src/components/Landing/Explore/index.js:18]: ahhhh gotcha. Well, perfect. That's fixed by the new grid change. Thank you!

**Comment by seungpark** [comment]
these changes LGTM 🚀

**Review by schmalliso — APPROVED** [review]

**Pull request merged**